src/csharp/Grpc.Core/Server.cs

```diff
@@ -443,7 +451,7 @@ internal ServerPortCollection(Server server)
             /// <summary>
             /// Adds a new port on which server should listen.
             /// Only call this before Start().
-            /// <returns>The port on which server will be listening.</returns>
+            /// <returns>The port on which server will be listening. Return value of zero means that binding the port has failed.</returns>
```

Comment:
Minor: there's the [`<returns>`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/returns) tag for return values. Not sure what the style for this code base is.

Reply:
Looks like that's what's being used.

Reply:
Reading is hard. ;-)